src/utils.ts

```diff
@@ -256,7 +256,11 @@ export function escObjKey(input: string): string {
 
 /** Indent a string */
 export function indent(input: string, level: number) {
-  return "  ".repeat(level).concat(input);
+  if (level > 0) {
+    return "  ".repeat(level).concat(input);
+  } else {
+    return input;
```

Comment:
Minor change to `indent()`—this isn’t needed for valid schemas, but for invalid schemas this often is the thing to throw an error and it seems broken.